### ISSUE TITLE
Generate a list of all websites on rbind automatically

### ISSUE THREAD

**Issue by yihui** [rbind/support#38]
Get JSON from https://api.github.com/users/rbind/repos, obtain `homepage` URLs, screenshot them (e.g. `webshot::webshot()`), and generate a showcase page automatically.

To to be able to do this, we need everybody to set their homepage URLs in their repos. Some have set the URL, and some have not.

Click the Edit button in your repo and fill out the URL:

![image](https://user-images.githubusercontent.com/163582/27454884-16fde460-5761-11e7-826c-85d4047cda3e.png)

There are many blogdown-based websites that are not on rbind yet, and I'll need someone to dig them out from https://t.yihui.name (manually or semi-automatically).

Also see http://awesome-blogdown.com/

**Comment by yihui** [comment]
Definitely. I almost always need assistance! Some people have not set their homepage URLs, and you need to ask them to do so first. Thanks!

**Comment by statimatics** [comment]
Hello Yihui,
I am willing to volunteer by putting in a bit of my time with the rbind project. Specifically, I am willing to:

1. I am wiling to hunt down new blogdown sites and invite them to join and transfer sites to rbind. I just transferred mine now despite setting up since Aug 2017 just because I didn't know. 
2. Help new users in setting up in common issues like custom domain and DNS records setup,  and redirects as I've noticed these are common issues for new users, like the type in #145.

In case you need additional helping hands with these. I see this as a way getting more experience with the blogdown package. In the mean time, kindly make me the admin of my newly transferred site: statimatics. Thanks.

**Comment by yihui** [comment]
@statimatics Excellent! I just made you the admin of this organization. Your help will be truly appreciated!

BTW, after people transfer their repositories to rbind, they can actually fork them to their personal accounts. That is what I do myself: my "official" repo is rbind/yihui, but I have a copy at yihui/yihui.name. Below is the part of my git config that makes it possible for me to push to the two repositories at the same time:

```ini
[remote "origin"]
        url = git@github.com:rbind/yihui.git
        url = git@github.com:yihui/yihui.name.git
        fetch = +refs/heads/*:refs/remotes/origin/*
```

That is just FYI, in case anyone ask about it. I guess most users probably won't care.

**Comment by thercast** [comment]
I had no idea it was possible to accomplish a push to multiple repos at once.  Thanks for sharing!

**Comment by road2stat** [comment]
Closing the issue for now and moving the discussions with project planning to https://github.com/rbind/support/projects/1.

**Comment by petrkajzar** [comment]
Hi, I like your project and your idea of creating an automatic list of all pages under the rbind.io domain. Getting a list from the GitHub API is quite simple, but most repos there do not provide information about the homepage. So for start I used a list of closed issues here and using the @apifytech crawler I got information about the original website domain (mostly on Netlify) and the new subdomain on rbind.io. This creates a "dirty" raw data, but Apify should be able to use automatic headless browser to make automatic screenshots and store them in a predefined database. I am definitely not so advanced user of Apify, I'm just trying the way. :-) See the [raw csv data](https://gist.github.com/petrkajzar/d1825bdaaca39bb98fc161352361c591) and [crawler homepage](https://www.apify.com/petrkajzar/q9hD6-api-github-com).
